shaka/src/public/ShakaPlayerView.mm

```diff
@@ -300,4 +308,28 @@ - (BOOL)remakeTextCues:(BOOL)sizeChanged {
   return YES;
 }
 
+- (void) orientationChanged:(NSNotification *)note {
+  if (_imageLayer.contents == nil) {
+    return;
+  }
+  // Fit image in frame.
+  shaka::ShakaRect<uint32_t> image_bounds = {0, 0, CGImageGetWidth(_image),
+                                                   CGImageGetHeight(_image)};
+  shaka::ShakaRect<uint32_t> dest_bounds = {
+    0,
+    0,
+    static_cast<uint32_t>(self.superview.bounds.size.width),
+    static_cast<uint32_t>(self.superview.bounds.size.height),
+  };
+  shaka::Rational<uint32_t> aspect_ratio;
```

Comment:
This value is filled from the frame in `_player.videoRenderer->Render`.  Passing an empty value won't always produce the correct result.  You'll need to store this too after getting a successful call to `Render`.

Reply:
Done